pkgs/development/idris-modules/default.nix

```diff
@@ -6,7 +6,7 @@
     overrideScope = f: callPackageWithScope (mkScope (fix' (extends f scope.__unfix__))) drv args;
   };
 
-  mkScope = scope : pkgs // pkgs.xorg // pkgs.gnome // scope;
```

Comment:
Maybe it was copy'n'paste from [haskell](https://github.com/NixOS/nixpkgs/commit/5898c2060433d803865df3a9af4408d0443de8b8).
In haskell there was probably a namespace conflict between gnome and a haskell package.